core/templates/pages/exploration-player-page/learner-experience/conversation-skin.component.ts

```diff
@@ -456,7 +458,8 @@ export class ConversationSkinComponent {
       if (
         !this.isIframed &&
         !this._editorPreviewMode &&
-        !this.explorationPlayerStateService.isInQuestionPlayerMode()
+        !this.explorationPlayerStateService.isInQuestionPlayerMode() &&
+        !this.explorationPlayerStateService.isInDiagnosticTestPlayerMode()
```

Comment:
The diagnostic test player is also a type of question player mode with modifications (like answer correctness is not shown), so this mode does not have the context of exploration, thus added here in the if clause.